ecs-cli/modules/clients/aws/ecs/client.go

```diff
@@ -166,17 +167,20 @@ func (c *ecsClient) CreateService(serviceName, taskDefName string, loadBalancer
 	return nil
 }
 
-func (c *ecsClient) UpdateServiceCount(serviceName string, count int64, deploymentConfig *ecs.DeploymentConfiguration) error {
-	return c.UpdateService(serviceName, "", count, deploymentConfig)
+// TODO: needs network config also?
+func (c *ecsClient) UpdateServiceCount(serviceName string, count int64, deploymentConfig *ecs.DeploymentConfiguration, networkConfig *ecs.NetworkConfiguration) error {
```

Comment:
it'd be awesome if all of the exported fields and methods had comments accompanying them

Reply:
Will add it to our backlog.